api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -122,4 +126,23 @@ public String getDate(List<FieldValue> row, int index) {
         }
         return row.get(index).getStringValue();
     }
+
+    public FieldList getTableFieldsFromDomain(Domain d) {
+        CdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
+        String tableName;
+        if (d.equals(Domain.CONDITION)) {
```

Comment:
nit: you can reverse these to avoid NullPointerExceptions since the equals method is null safe. 
```suggestion
        if (Domain.CONDITION.equals(d)) {
```